src/config.ts

```diff
@@ -5,6 +5,7 @@ import { ListSettings, ListViews, Pagination } from "./types";
 export const APP_MOUNT_URI = process.env.APP_MOUNT_URI;
 export const APP_DEFAULT_URI = "/";
 export const API_URI = process.env.API_URI;
+export const SW_INTERVAL = parseInt(process.env.SW_INTERVAL, 0);
```

Comment:
passing `0` is equal to skipping the argument entirely
```
export const SW_INTERVAL = parseInt(process.env.SW_INTERVAL);
```

Reply:
Yup but linter screams